.zuul.yaml

```diff
@@ -32,6 +35,7 @@
         override-checkout: main
       - name: github.com/openstack-k8s-operators/telemetry-operator
         override-checkout: main
+    timeout: 18000  # 5 hours
```

Comment:
Is this planed to stay? Do we expect some significant extension of how long the jobs run?

Reply:
No, I think it can be removed.
There were some timeouts initially, caused by autoscaling not being deployed, so I extended the timeout to get some results and errors to help debug.